Gemfile

```diff
@@ -34,7 +34,7 @@ gem 'devise_token_auth', '~> 1.0'
 gem 'active_model_serializers'
 gem 'bootsnap', require: false
 gem 'image_processing', '~> 1.2'
-gem 'aws-sdk-s3', require: false
+gem 'aws-sdk-s3', '~> 1.48.0', require: false
```

Comment:
Why do we need to specify this version? Fine if we do.

Reply:
Oops, that's actually from https://github.com/performant-software/dm-2/pull/299 and I branched this off of that branch.